<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Talletus</class>
 <widget class="QDialog" name="Talletus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="textSumma">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;  border-width: 2px;   border-color: rgba(255, 140, 40,1);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonEnter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#buttonEnter{
background-color:rgba(242, 140, 40,1);
color:  white;
}
QPushButton#buttonEnter:hover{
background-color:rgba(230, 120, 40,0.9); color: rgba(0, 0, 0, 0.8); 
}</string>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(242, 140, 40,0.8);
color: white;</string>
   </property>
   <property name="text">
    <string>Syot√§ summa</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="peruutaButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#peruutaButton{
background-color:rgba(242, 140, 40,1);
color:  white;
}
QPushButton#peruutaButton:hover{
background-color:rgba(230, 120, 40,0.9); color: rgba(0, 0, 0, 0.8); 
}</string>
   </property>
   <property name="text">
    <string>Peruuta</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
